csharp/ql/test/query-tests/Security Features/CWE-601/UrlRedirect/UrlRedirect.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Web;
+using System.Web.Helpers;
```

Comment:
According to https://learn.microsoft.com/en-us/dotnet/api/system.web.webpages.requestextensions.isurllocaltohost?view=aspnet-webpages-3.2, the method lives in `System.Web.WebPages`.

Reply:
Yes, you are right - I will fix that.